packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -77,22 +77,17 @@ export enum EmailEventStatusEnum {
   BLOCKED = 'blocked',
   SPAM = 'spam',
   UNSUBSCRIBED = 'unsubscribed',
-  SPAM_COMPLAINED = 'spam complained',
-  SUBSCRIPTION_CHANGED = 'subscription changed',
 }
 
 export enum SmsEventStatusEnum {
   CREATED = 'created',
   DELIVERED = 'delivered',
-  RECEIVED = 'received',
```

Comment:
Hey, I have a small question why did we remove the additional event statuses with their mapping?

Reply:
Because we are not interested in knowing that some one sent an sms to our Novu user :)